internal/test_unit/xml/unit_plot_data_plane.xml

```diff
@@ -694,4 +694,34 @@
     </output>
   </test>
 
+  <test name="plot_data_plane_WRF_fire_subgrid">
+    <exec>&MET_BIN;/plot_data_plane</exec>
+    <env>
+      <pair><name>MET_USE_WRF_SUBGRID</name> <value>yes</value></pair>
+    </env>
+    <param> \
+      &DATA_DIR_MODEL;/wrf/wrfout_fire_d02_2018-06-01_20:00:00 \
+      &OUTPUT_DIR;/plot_data_plane/wrf_fire_subgrid.ps \
+      'name="FIRE_AREA";level="0,*,*";' \
```

Comment:
```suggestion
      'name="FIRE_AREA"; level="(0,*,*)";' \
```
By convention, we enclose all the other level strings for NetCDF inputs in parenthesis. Recommend consistently using them here to avoid confusion.